<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>566</width>
    <height>546</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>System Monitor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton_reboot">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>450</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;pkill&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">pkill
</string>
    </property>
    <property name="text">
     <string>Reboot</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_halt">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>450</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Shutdown</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget_INFO">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>531</width>
      <height>441</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tabWidget_raminfo">
     <attribute name="title">
      <string>Sources</string>
     </attribute>
     <widget class="QLabel" name="label_SWAP_Free">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>349</y>
        <width>66</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>0MB</string>
      </property>
     </widget>
     <widget class="QListView" name="listView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>511</width>
        <height>171</height>
       </rect>
      </property>
     </widget>
     <widget class="QListView" name="listView_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>190</y>
        <width>511</width>
        <height>201</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_SWAP_Total">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>350</y>
        <width>66</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>8048MB</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_RAM_Free">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>260</y>
        <width>66</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>2500MB</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_RAM_Total">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>260</y>
        <width>66</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>7843MB</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>200</y>
        <width>66</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Memory</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>66</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>CPU</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>261</y>
        <width>41</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Free</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>350</y>
        <width>41</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Free</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>260</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Total</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>350</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Total</string>
      </property>
     </widget>
     <widget class="QwtPlot" name="qwtplot_cpu">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>331</width>
        <height>121</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>290</y>
        <width>66</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Swap</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_cpuinfo">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>80</y>
        <width>41</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>5%</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>80</y>
        <width>101</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>CPU frequency</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBar_RAM">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>230</y>
        <width>351</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBar_SWAP">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>310</y>
        <width>351</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
     <zorder>listView_2</zorder>
     <zorder>label_SWAP_Free</zorder>
     <zorder>listView</zorder>
     <zorder>label_SWAP_Total</zorder>
     <zorder>label_RAM_Free</zorder>
     <zorder>label_RAM_Total</zorder>
     <zorder>label</zorder>
     <zorder>label_2</zorder>
     <zorder>label_15</zorder>
     <zorder>label_16</zorder>
     <zorder>label_19</zorder>
     <zorder>label_20</zorder>
     <zorder>qwtplot_cpu</zorder>
     <zorder>label_17</zorder>
     <zorder>label_cpuinfo</zorder>
     <zorder>label_18</zorder>
     <zorder>progressBar_RAM</zorder>
     <zorder>progressBar_SWAP</zorder>
    </widget>
    <widget class="QWidget" name="tabWidget_INFOPage2">
     <attribute name="title">
      <string>Process</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_pkill">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>370</y>
        <width>98</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>结束进程</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_prefresh">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>370</y>
        <width>98</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>刷新</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_pNum">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>10</y>
        <width>71</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_pRun">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>10</y>
        <width>66</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_pSleep">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>40</y>
        <width>66</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_pZombie">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>40</y>
        <width>66</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>10</y>
        <width>66</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Total</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>40</y>
        <width>81</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Sleeping</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>10</y>
        <width>81</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Running</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>40</y>
        <width>81</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Zombie</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget_process">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>80</y>
        <width>471</width>
        <height>281</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>Pid</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>State</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Priority</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Memory(kb)</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="tabWidget_INFOPage3">
     <attribute name="title">
      <string>Module</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_Model_refresh">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>370</y>
        <width>98</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>刷新</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget_model">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>471</width>
        <height>341</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Memory (kb)</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Used Times</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Network</string>
     </attribute>
     <widget class="QListWidget" name="listWidget_devspeed">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>491</width>
        <height>111</height>
       </rect>
      </property>
     </widget>
     <widget class="QListWidget" name="listWidget_tcpspeed">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>180</y>
        <width>491</width>
        <height>201</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabWidget_INFOPage4">
     <attribute name="title">
      <string>System</string>
     </attribute>
     <widget class="QListView" name="listView_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>511</width>
        <height>211</height>
       </rect>
      </property>
     </widget>
     <widget class="QListView" name="listView_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>250</y>
        <width>511</width>
        <height>141</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_CPUName">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>60</y>
        <width>311</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_CPUType">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>100</y>
        <width>291</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_SystemType">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>300</y>
        <width>251</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_SystemVersion">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>340</y>
        <width>261</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>60</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>CPU Version</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>100</y>
        <width>71</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>CPU Type</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>300</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>OS Type</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>340</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>OS Version</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>260</y>
        <width>111</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>OS Information</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>121</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>CPU Information</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_CatheCapacity">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>180</y>
        <width>291</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>180</y>
        <width>81</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Cache size</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>140</y>
        <width>111</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>CPU Cores</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_CPUCores">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>140</y>
        <width>291</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>566</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header>qwt_plot.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
